### ISSUE TITLE
Make use of the new Accent Color Option in Millennium

### ISSUE THREAD

**Issue by GrPK** [SaiyajinK/Minimal-Dark-for-Steam#31]
Hey! The Millennium Patcher can now have a custom accent color. Is there any way to make use of it in your theme? Thanks!

**Comment by SaiyajinK** [comment]
Hi,
Honestly I don't know what this section corresponds to, if it is feasible or not. I asked the question to the dev of millenium I am waiting for his return before deciding 👍🏻

**Comment by SaiyajinK** [comment]
@GrPK hi,

Sorry for the wait but I redirected to the millennium dev but I didn't get any answer, so I had to search on my side with my only (limited) knowledge.

I think I have a start of a lead:

![steamwebhelper_Zn1eJ8kvsh](https://github.com/user-attachments/assets/9ba7d89d-9166-4f29-8ed4-81ad6a17a524)

The default color will be the Windows accent color but the latter will also be customizable from the Millennium color selector. When you click on RESET it puts back the default Windows accent color.

the only problem is that the selected color does not apply at all on the webkit, it may be normal I don't know but in any case it does not work and I have to put a static color (blue by default).

I am still working on the question I do not know when I will make this available but it is already an encouraging start.
